src/ferry_planner/schedule.py

```diff
@@ -24,9 +25,11 @@ class FerrySailing(BaseModel):
     """Duration in seconds."""
     # TODO: price: float  # noqa: FIX002
     """Price in Canadian dollars (CAD)."""
+    notes: Sequence[str] | None
```

Comment:
```suggestion
    notes: tuple[str, ...] = ()
```
A `None` value has no special meaning here

Same applies to `FerrySchedule` and `HtmlParseResult`